ddnoise.js

```diff
@@ -88,14 +87,12 @@ define(['./utils', 'underscore', 'promise'], function (utils, _) {
         if (this.state === SPINNING || this.state === SPIN_UP) return;
         this.state = SPIN_UP;
         var self = this;
-        var promise = self.play(self.sounds.motorOn);
-        if (promise === null) return;
-        promise.then(function () {
+        this.play(this.sounds.motorOn).then(function () {
             self.play(self.sounds.motor, true).then(function (source) {
                 self.motor = source;
                 self.state = SPINNING;
             });
-        });
+        }, function () {});
```

Comment:
I'm not sure what the second function is here for? Are you looking to catch exceptions? (I think that needs a `.catch()...` ) Also it's early in the morning here so I'm probably missing something again hahah!

Thanks for tolerating my inane comments..

Reply:
It's interesting, .catch() also works to resolve the problem in a quick test.

What is happening here is that when you reject a promise and the then() call doesn't have a 2nd argument (the rejection function), the Chrome Javascript console registers an exception in an sync context. I think it's harmless but it seems like a wart we could do without.

I think I prefer to keep the solution that provides an empty rejection function. This is because what we're trying to do here is handle a clean rejection by just moving along, rather than catch any exceptions. This way, we'll also get clear exception details in the console if something _does_ start excepting at a later date.

Reply:
That should be "registers an exception in an _async_ context"

Reply:
I'll abuse the approval to merge this as-is for now -- if you prefer catch() I'm happy to update to it in a subsequent PR.

Reply:
Oh totally! No worries: this is just great as it is! :) You have full commit rights, I appreciate the PRs but I trust you to just make changes if that makes sense too :)